### DISCUSSION TITLE
Getting Started with Mesmerizer

### DISCUSSION THREAD

**Discussion by hevnsnt** [PlummersSoftwareLLC/NightDriverStrip#531] [General]
Hi all, I recently moved and got a notice that I have a package to pick-up (from awhile ago) and found a Mesmerizer waiting for me!  Is there any documentation of this device, so I can get started working on it?  

Is there any way to extend it to other LED panels (I have several of the same board laying around)

**Comment by robertlipe** [comment]
Welcome.

There's a wealth of information in the *.md files at
https://github.com/PlummersSoftwareLLC/NightDriverStrip as well as a number
of success (and qualified success) stories collected in our history at
https://github.com/PlummersSoftwareLLC/NightDriverStrip/discussions

In theory, multiple panels can be supported at the same time, but I think
the few people that have tried it said we run out of bandwidth beyond about
3 panels on the HUB75 bus. Just the reality of a couple of panels worth of
2k or 4k pixel panels of 24-bit colors at a reasonable frame rate of
animation combined with the HUB75's design of pushing refresh to the bus
master means you're just not going to push a huge number of panels around
with these.

If you have 64x64 panels, be sure to see the recent discussions at
https://github.com/PlummersSoftwareLLC/NightDriverStrip/discussions/526 and
the inner-linked discussion.

I look forward to more people exploring the bounds of such things.
